### ISSUE TITLE
ActiveRecord::Tasks::DatabaseNotSupported with Rails 5.0.0.1 after upgrade to 4.0.1

### ISSUE THREAD

**Issue by filiptepper** [rgeo/activerecord-postgis-adapter#238]
```bash
rake db:migrate
D, [2016-11-09T08:17:02.203542 #19048] DEBUG -- :    (0.4ms)  SELECT pg_try_advisory_lock(5271997261597374000);
D, [2016-11-09T08:17:02.229106 #19048] DEBUG -- :   ActiveRecord::SchemaMigration Load (1.1ms)  SELECT "schema_migrations".* FROM "schema_migrations"
D, [2016-11-09T08:17:02.254637 #19048] DEBUG -- :   ActiveRecord::InternalMetadata Load (0.8ms)  SELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2  [["key", :environment], ["LIMIT", 1]]
D, [2016-11-09T08:17:02.261272 #19048] DEBUG -- :    (0.4ms)  BEGIN
D, [2016-11-09T08:17:02.263314 #19048] DEBUG -- :    (0.4ms)  COMMIT
D, [2016-11-09T08:17:02.264097 #19048] DEBUG -- :    (0.5ms)  SELECT pg_advisory_unlock(5271997261597374000)
rake aborted!
ActiveRecord::Tasks::DatabaseNotSupported: Rake tasks not supported by 'postgis' adapter
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/tasks/database_tasks.rb:281:in `class_for_adapter'
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/tasks/database_tasks.rb:207:in `structure_dump'
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/railties/databases.rake:292:in `block (3 levels) in <top (required)>'
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/railties/databases.rake:67:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/railties/databases.rake:59:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:structure:dump
(See full trace by running task with --trace)
```

Works fine with 4.0.0.

**Comment by prikha** [comment]
Can confirm that. I believe that there was only one change affecting it: 

https://github.com/rgeo/activerecord-postgis-adapter/commit/808e89c5458f83a859a188403cc84662b43e40f9

**Comment by teeparham** [comment]
Caused by #237

**Comment by teeparham** [comment]
Fixed by 4bbe4fe, released as version 4.0.2
